### DISCUSSION TITLE
用 NDM 下载视频时提示 HTTP/1.1 403 Forbidden

### DISCUSSION THREAD

**Discussion by SunJunge08** [the1812/Bilibili-Evolved#3283] [提问 / Q&A] [answered]
输出 IDM 的 `.ef2` 文件后，把 User-Agent 复制到 NDM 设置的“默认用户代理”中。接着把下载链接添加到 NDM 中，然后就会提示“下载错误”，界面如下。

![image](https://user-images.githubusercontent.com/70008772/165754416-787d04bc-49a6-4f1a-845f-701c2bef8fa4.png)

这个问题应该怎么解决？

**Comment by the1812** [comment]
把 Referer 设为视频页面的网址

**Reply by SunJunge08** [reply]
怎么设置？NDM 没有这个选项。

**Reply by the1812** [reply]
这得你自己研究了，我没用过 NDM

**Comment by SunJunge08** [comment, marked as answer]
已经放弃 NDM 了，还是 IDM 好用。

**Reply by MeowxYee** [reply]
NDM是有这个设置的，就在你的主面板中有。
![image](https://user-images.githubusercontent.com/18101168/185775869-b3f6d6cf-e3e2-4380-a923-e074f64fd215.png)

**Comment by jesee030** [comment]
换UA啊.easy

**Reply by SunJunge08** [reply]
> 换UA啊.easy

![image](https://user-images.githubusercontent.com/70008772/178490755-6d6542bf-8dd7-4f6f-995c-3495f1d26d5a.png)

你有没有好好看我的描述？我已经填写 UA 了。

**Reply by jesee030** [reply]
填了和能用是两回事

**Reply by SunJunge08** [reply]
> 填了和能用是两回事

那你给我一个能用的UA

**Reply by jesee030** [reply]
https://kinhdown.com/?Type=Tutorials

**Reply by SunJunge08** [reply]
> https://kinhdown.com/?Type=Tutorials

什么意思？

**Reply by jesee030** [reply]
或者你下个插件，类似ua-Switch这类
